tests/test_application.py

```diff
@@ -201,6 +201,18 @@ async def post_shutdown(application: Application) -> None:
                 post_shutdown=None,
             )
 
+    def test_job_queue(self, bot, app, recwarn):
+        expected_warning = (
+            "No `JobQueue` set up. To use `JobQueue`, you must install PTB via "
+            "`pip install python-telegram-bot[job_queue]`."
```

Comment:
There's a lot of repetition of this warning message, so I usually have the urge to extract it into a constant, but I know there's some skepticism about this, especially in tests.